servicecontrol/creating-config-file.md

```diff
@@ -257,6 +257,9 @@ Type: string
 
 Default: `<ErrorQueue>.log`
 
+Version 1.28 and below created the queue specified by this setting when the service instance is installed regardless of the value of `ServiceControl/ForwardErrorMessages`.  From V1.29 the queue is only created if `ServiceControl/ForwardErrorMessages` is enabled. 
```

Comment:
I don't think the current text is ambiguous which is what version language guideline is trying to prevent.  Happy to change it though if you think it is